README.md

```diff
@@ -98,7 +98,42 @@ To do this, include flags from only one of the two options below, only use one o
 - To specify the number of training splits and test splits directly, use the three flags `--num_training_splits=...`, `--num_dev_splits=...` and `--num_test_splits=...`
 - To specify the percentage of the data heldout for testing, you can specify `--dev_ratio=...` and `--test_ratio=0.1`, where 0.1 means that approximately 10% of the data will be included in the test splits. You can also specify the `--num_training_splits=...` flag to control the total number of training splits, but we recommend to let this default.
 
-### Flags
+### Dataset Size Requirements
+When preparing a dataset for training, different dataset sizes will dictate the maximum batch size you can set for training. It is *__necessary__* to know this maximum batch size so you can set it accordingly for your training job. 
+
+#### How to Check and Set
```

Comment:
Tried to lead with this section since it's direct to the point, allowing the beginner to get the pertinent info without having to wade through a full explanation until they need it. Happy to hear opinions on structure if you think there's a better way though!

Reply:
Looks good